tests/efs-check.sh

```diff
@@ -2,50 +2,45 @@
 
 set -e
 
-paths=${1}
+path=${1}
 fs_id=${2}
-paths=$(echo ${paths} | tr "," " ")
 status=0
 
-pathCount=$(cat /etc/fstab | grep -E "$(echo ${paths} | tr " " "|")" | wc -l | xargs)
-fsGrep=$(cat /etc/fstab | grep -E "fs-" | wc -l | xargs)
-truePathCount=$(echo ${paths} | tr " " "\n"| wc -l |xargs)
+path_exist=$(cat /etc/fstab | grep "$(echo ${path})";echo $?)
+fs_grep=$(cat /etc/fstab | grep "fs-";echo $?)
```

Comment:
Why are you relying on $? instead of if -z $fs_grep?

Reply:
I had to do echo $? because if grep does not find a match it will not return an empty string but rather it will exit 1 and the shell script will stop right on the failure, so I had to do that in order to keep the script running and print error messages.